### PR TITLE
fix phpdoc and phpStorm other recommandations

### PR DESCRIPTION
**Pull request by tcitworld**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Documentation | no |
| Translation | no |
| Fixed tickets |  |
| License | MIT |

### CODE REVIEW

File: src/Wallabag/CoreBundle/Command/InstallCommand.php [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 namespace Wallabag\CoreBundle\Command;
 
+use Doctrine\DBAL\Connection;
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Command/InstallCommand.php]: This is wrong, my bad.

File: src/Wallabag/AnnotationBundle/Controller/WallabagAnnotationController.php [hunk 2]
```diff
@@ -21,6 +21,7 @@ class WallabagAnnotationController extends FOSRestController
      *      }
      * )
      *
+     * @param Entry $entry
      * @return Response
```
Comment (j0k3r) [inline src/Wallabag/AnnotationBundle/Controller/WallabagAnnotationController.php]: Could you please put an extra line between each `@param` & `@return`?

File: src/Wallabag/CoreBundle/Event/Subscriber/CustomDoctrineORMSubscriber.php [hunk 3]
```diff
@@ -12,13 +13,12 @@
  */
 class CustomDoctrineORMSubscriber extends DoctrineORMSubscriber implements EventSubscriberInterface
 {
-    /**
-     * @param GetFilterConditionEvent $event
-     */
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Event/Subscriber/CustomDoctrineORMSubscriber.php]: :no_good_man:

File: src/Wallabag/CoreBundle/Form/DataTransformer/StringToListTransformer.php [hunk 4]
```diff
@@ -44,12 +44,12 @@ public function transform($list)
...
      * @param string $string
      *
-     * @return array|null
+     * @return null|array
      */
     public function reverseTransform($string)
     {
         if ($string === null) {
-            return;
+            return null;
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Form/DataTransformer/StringToListTransformer.php]: 👎 `return;` is equivalend to `return null;`

**Pull request closed without merging**